scripts/ddl2cpp

```diff
@@ -181,7 +182,11 @@ for create in tableCreations:
         print('            const T& operator()() const { return ' + columnMember + '; }', file=header)
         print('          };', file=header)
         print('      };', file=header)
-        traitslist = [NAMESPACE + '::' + types[sqlColumnType]];
+        try:
+            traitslist = [NAMESPACE + '::' + types[sqlColumnType]];
+        except KeyError as e:
+            print ('Error: datatype "'  + sqlColumnType + '"" is not supported. Implement it (examples: sqlpp11/data_types) or raise an issue on github')
```

Comment:
how?
(this patch is a quick fix for someone who stumbles upon sqlpp11 and tries to use ddl2cpp, so when he got an exception he knows it's not a bug or bad code in ddl2cpp)